src/components/removebookbutton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+function RemoveBookButton({ id }) {
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(removeBook(id));
+  };
+
+  return (
+    <button type="button" onClick={handleClick}>Remove Book</button>
+  );
```

Comment:
[OPTIONAL] - You can consider using `type="submit"` on the submit button so that when the enter key is pressed, the form will submit, improving the user's experience of your website. 👍